src/components/sidebar/components/sidebar-mobile-controls/index.tsx

```diff
@@ -70,10 +70,19 @@ const LevelDownButton = ({ onClick, text }: ButtonProps) => {
 	)
 }
 
-interface SidebarMobileControlsProps {
-	levelUpButtonText: ButtonProps['text']
-	levelDownButtonText: ButtonProps['text']
-}
+/**
+ * One or both of `levelUp` and `levelDown`
+ * `ButtonText` properties must be provided.
+ */
+type SidebarMobileControlsProps =
```

Comment:
I wanted to use the `SidebarProps` type in `docs-view/server`, but needed to make it so either `levelUpButtonText` or `levelDownButtonText` or both could be provided here. This type aims to accomplish that.